pkg/pipeline/crd.go

```diff
@@ -49,6 +50,7 @@ type CRDGenerator struct {
 	GroupDir       string
 	RootModulePath string
 	Group          string
+	Category       string
```

Comment:
What do you think about `ProviderShortName`? `Category` feels closer to group, i.e. `ec2`, `rds` etc., but that's not actually what we mean, right?

Reply:
Done.

Reply:
Thanks!